public/components/notebooks/components/paragraph_components/para_output.tsx

```diff
@@ -100,9 +100,9 @@ const OutputBody = ({
    */
   const dateFormat = uiSettingsService.get('dateFormat');
   const from = convertDateTime(visInput?.timeRange?.from, true, false);
-  const to = convertDateTime(visInput?.timeRange?.from, false, false);
+  const to = convertDateTime(visInput?.timeRange?.to, false, false);
   const displayFrom = convertDateTime(visInput?.timeRange?.from, true) || 'Invalid date';
-  const displayTo = convertDateTime(visInput?.timeRange?.from, false) || 'Invalid date';
+  const displayTo = convertDateTime(visInput?.timeRange?.to, false) || 'Invalid date';
```

Comment:
I see the date format not being used here. We need to use the specific date-format provided by user in the dashboards settings. This helps to parse absolute dates and formats the way user wants to see it. 
2.11 implementation: https://github.com/opensearch-project/dashboards-observability/blob/920fa00b9ad3579c709f3544cb7fd319210c944c/public/components/notebooks/components/paragraph_components/para_output.tsx#L132-L135

Reply:
Has been restored.